inventory_management_system_api/schemas/catalogue_category.py

```diff
@@ -124,6 +127,16 @@ def check_valid_allowed_values(
                             "itself"
                         )
 
+                    # Ensure the value isn't duplicated
+                    seen_value = (
+                        allowed_value
+                        if catalogue_item_property_data["type"] != CatalogueItemPropertyType.STRING
+                        else allowed_value.lower()
+                    )
+                    if seen_value in seen_values:
```

Comment:
Could you please update the docstring of this method to mention this check?

Reply:
I moved it to the :raises section as I thought it might be clearer to list them as bullet points.